subsys/bluetooth/controller/Kconfig

```diff
@@ -47,6 +47,14 @@ config BT_CTLR_SYNC_PERIODIC_SUPPORT
 	depends on BT_CTLR_ADV_EXT_SUPPORT
 	bool
 
+config BT_CTLR_SYNC_TRANSFER_SENDER_SUPPORT
+	depends on BT_CTLR_SYNC_PERIODIC_SUPPORT
+	bool
+
+config BT_CTLR_SYNC_TRANSFER_RECIPIENT_SUPPORT
+	depends on BT_CTLR_SYNC_PERIODIC_SUPPORT
+	bool
+
```

Comment:
For controller configurations we typically have a set of two configurations:
1. A `BT_CTLR_<feature>_SUPPORT` config. This only indicates if the controller supports it, it does not include it in the build
2. A `BT_CTLR_<feature>` config. Setting this config will include it in the build. In your care you want to have something like
```
config BT_CTLR_SYNC_TRANSFER_SENDER
	bool "Periodic sync transfer sendder"
	depends on BT_CTLR_SYNC_TRANSFER_RECIPIENT_SUPPORT
	default BT_PER_ADV_SYNC_TRANSFER_SENDER
	help
	  Enable support for the periodic advertising sync transfer control procedure
	  as a transmitter of the LL_PERIODIC_SYNC_IND. See Core_v5.3, Vol 6, Part B,
	  Section 5.1.13
```
This config will not yet be used, but should be made available for controllers not present in vanilla zephyr builds.